docs/release/trg-0/trg-1-05.md

```diff
@@ -0,0 +1,35 @@
+---
+title: TRG 1.05 - Documentation Architecture
```

Comment:
Architecture Documentation instead of the other way around?

Reply:
TRG 1.05 - Architecture Documentation